mass_api_client/resources/scheduled_analysis.py

```diff
@@ -13,8 +14,8 @@ class ScheduledAnalysis(BaseResource):
     def create(cls, analysis_system_instance, sample):
         return cls._create(analysis_system_instance=analysis_system_instance.url, sample=sample.url)
 
-    def create_report(self, json_report_objects=None, raw_report_objects=None, tags=None):
-        return Report.create(self, json_report_objects=json_report_objects, raw_report_objects=raw_report_objects, tags=tags)
+    def create_report(self, analysis_date=datetime.datetime.now(), json_report_objects=None, raw_report_objects=None, tags=None):
```

Comment:
This will set the time once at import of the class. See:
```
>>> def sample(analysis_date=datetime.datetime.now()):
...     print(analysis_date)
... 
>>> sample()
2017-06-29 10:57:59.131380
>>> sleep(5)
>>> sample()
2017-06-29 10:57:59.131380
>>> 
```